ndex2/client.py

```diff
@@ -919,6 +1270,19 @@ def update_network_group_permission(self, groupid, networkid, permission):
         """
         Updated group permissions
 
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		groupid = 'INSERT USER ID HERE'
+		networkid = 'INSERT NETWORK UUID HERE'
+		#permission can be changed to other things
+		my_ndex.update_network_group_permission(groupid, networkid, permission='READ')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client